library/core/src/convert/mod.rs

```diff
@@ -715,6 +717,14 @@ impl fmt::Display for Infallible {
     }
 }
 
+#[cfg(not(bootstrap))]
+#[stable(feature = "str_parse_error2", since = "1.8.0")]
```

Comment:
should be convert_infallible?

Reply:
Mind fixing this up?

Reply:
I don't mind, but I'm very confused as to why it was marked as stable under the `str_parse_error2` feature in the first place...

https://github.com/rust-lang/rust/pull/99917/files#diff-6605ea65189fd18a801464d4718ad307bb93be455d1592603ebf9382063902b2L670

I think I'm going to do a little investigating first before I fix this up to make sure there wasn't a good reason.

Reply:
I figured out why https://github.com/rust-lang/rust/commit/ef485c6fa97a562b0bc559ffbe38fee9bf1f745b

Reply:
Ah! Nice catch. I think it's fine to keep as-is then.